apps/meteor/tests/e2e/utils/test.ts

```diff
@@ -13,15 +13,17 @@ const PATH_NYC_OUTPUT = path.join(process.cwd(), '.nyc_output');
 
 export type AnyObj = { [key: string]: any };
 
+export type BaseTestAPI = {
+	recreateContext(): Promise<void>;
+	login(credentials: { username: string; password: string }): Promise<APIRequestContext>;
+	get(uri: string, params?: AnyObj, prefix?: string): Promise<APIResponse>;
+	post(uri: string, data: AnyObj, prefix?: string): Promise<APIResponse>;
+	put(uri: string, data: AnyObj, prefix?: string): Promise<APIResponse>;
+	delete(uri: string, params?: AnyObj, prefix?: string): Promise<APIResponse>;
+}
+
```

Comment:
By following the first comment and removing the function to update the settings, we don't need this anymore